### ISSUE TITLE
Dead links

### ISSUE THREAD

**Issue by cuhe-la** [fatihacet/turkcekaynaklar-com#457]
Hi if you would like to fix/clean them, I'm sharing some dead links:
`curl https://raw.githubusercontent.com/fatihacet/turkcekaynaklar-com/master/README.md | grep -Po "(?<=\]\()[^\)]+(?=\))" | xargs -I % sh -c 'echo %; curl --connect-timeout 3 -Is % | head -n 1' | grep "HTTP/2 404" -B1`

```
https://github.com/vimevim/turkce-c-kaynaklari
HTTP/2 404 
--
https://github.com/vimevim/turkce-C-sharp-kaynaklari
HTTP/2 404 
--
https://github.com/vimevim/turkce-cpp-kaynaklari
HTTP/2 404 
--
https://emredurukn.github.io/2018/11/09/ruby-ile-web-scrapinge-giris.html
HTTP/2 404 
--
https://yusufozgul.com/sign-in-with-apple-nasil-kullanilir/
HTTP/2 404 
--
https://yusufozgul.com/swift-uimotioneffect-kullanimi/
HTTP/2 404 
--
https://yusufozgul.com/swiftuia-ilk-bakis/
HTTP/2 404 
--
https://cliexperiences.wordpress.com/2017/08/01/whatsapp-altyapisina-teknik-bir-bakis/
HTTP/2 404 
--
https://www.spreaker.com/show/parasutle-uretim-bandi
HTTP/2 404 
--
https://podcasts.apple.com/tr/podcast/turuncu-pasaport/id1435681904
HTTP/2 404 
--
https://www.facebook.com/izmirdev/
HTTP/2 404 

```

**Comment by cuhe-la** [comment]
`curl http://archive.org/wayback/available?url=https://cliexperiences.wordpress.com/2017/08/01/whatsapp-altyapisina-teknik-bir-bakis/ | jq -r '.archived_snapshots.closest.url' `
Code snippet might be useful to find available snapshot, if there is any.

**Comment by AtahanKocc** [comment]
**_I'm sharing angular dead links_;**
http://www.ilkaygunel.com/blog/2017/angular-2-tutorials/ 
https://www.yazilimgunlugu.org/category/javascript-web-uygulama-gelistirme/angular-2-4-turkce/

**Comment by KaganCanSit** [comment]
I've created a merge request for the issues listed here. You can assign this isseu to me. If there are additional improvements, I would like to help.

Best regards.

**Comment by KaganCanSit** [comment]
@gokaygurcan I think this issue can be closed.

**Comment by gokaygurcan** [comment]
Fixed in #486
